agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -590,23 +590,9 @@ bool main_thread::backhaul_fsm_main(bool &skip_select)
                 // Mark the connection as WIRED
                 m_sConfig.eType = SBackhaulConfig::EType::Wired;
 
-                LOG(DEBUG) << "add wired_backhaul to bridge " << m_sConfig.wire_iface;
-                if (!network_utils::linux_add_iface_to_bridge(m_sConfig.bridge_iface,
```

Comment:
This function can now also be removed from network_utils.

However, we may need to revive this code for backhaul optimisation (wired/wireless) so perhaps it's better to keep it in network_utils for the time being.

Reply:
This should not be simply removed but replaced with a check - and trigger slave reset if the wired interface is not in the bridge IMO.

Reply:
In addition, wired_backhaul option in configuration file used to mark that we either use the WAN interface for wired backhaul (if enabled) OR as a normal LAN interface. Since we are in passive mode, this should be taken care of by the platform, so this option should also be removed.

Reply:
Added check if the iface is in the bridge.

Reply:
I will remove `wired_backhaul` option from bpl.